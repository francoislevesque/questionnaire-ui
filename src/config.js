export default {
    url: 'http://questionnaire.test/api' // TODO : Change this URL for production
}
