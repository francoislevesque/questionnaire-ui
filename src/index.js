export { default as Questionnaire } from './components/Questionnaire';
