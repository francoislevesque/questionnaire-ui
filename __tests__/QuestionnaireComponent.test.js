import { Questionnaire } from '../src';
import Vue from 'vue/dist/vue.js';

describe('Questionnaire', () => {
    Vue.component('questionnaire', Questionnaire);

    beforeEach(() => {
        document.body.innerHTML = `
            <div id="app">
                <questionnaire></questionnaire>
            </div>
        `;
    });

    it('can mount', async () => {
        await createVm();

        expect(document.body.innerHTML).toMatchSnapshot();
    });
});

async function createVm() {
    const vm = new Vue({
        el: '#app',
    });

    await Vue.nextTick(() => {});

    return { app: vm, component: vm.$children[0] };
}
