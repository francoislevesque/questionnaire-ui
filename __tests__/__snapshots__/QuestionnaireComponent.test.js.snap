// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Questionnaire can mount 1`] = `
"
            <div id=\\"app\\"><div class=\\"questionnaire\\"> <!----></div></div>
        "
`;
